Content.Shared/Strip/SharedStrippableSystem.cs

```diff
@@ -118,6 +118,9 @@ private void StripHand(
             !Resolve(target, ref targetStrippable))
             return;
 
+        if (!target.Comp.CanBeStripped)
+            return;
+
```

Comment:
Should such a check maybe be added to CanStripRemoveHand as well? I think works without in our codebase but maybe some forks use that function for something else.

Reply:
I guess it can't hurt.